src/app/interfaces/storage.interface.ts

```diff
@@ -0,0 +1,75 @@
+// As returned by pool.query under topology[<vdevtype>]
+export interface VDev {
+  'type': string;
+  'path': string;
+  'guid': string;
+  'status': string;
+  'children': this[];
+  'device'?: string;
+  'disk'?: string;
+  'unavail_disk': any;
+  'stats': VDevStats;
```

Comment:
You sure linter is happy about this? :)
I think it should have removed quotes around interface keys.

Reply:
No it's not. I think when I imported some files from stable branch it clobbered some of the linter's changes. I noticed in the diff an enum also got wiped out.